client/gefyra/types.py

```diff
@@ -178,7 +178,8 @@ def get_client_config(
                 ca_crt=self.service_account["ca.crt"],
                 gefyra_server=gefyra_server,
                 registry=registry,
-                wireguard_mtu=str(wireguard_mtu),
```

Comment:
This problem also exists everywhere where the `ClientConfiguration` object is instantiated. The typing indicates that this is a problem.
![Bildschirmfoto 2025-02-17 um 17 55 44](https://github.com/user-attachments/assets/7d387d5f-10f8-4aa8-b5f7-c9c019798b06)

It results in this error:
![image](https://github.com/user-attachments/assets/8e7e80d2-b641-4e39-93d9-64e9be48f507)

Please check the passing of these parameters for all `ClientConfiguration` inits.